### DISCUSSION TITLE
ripristino backup HA e DB 0 Mb

### DISCUSSION THREAD

**Discussion by calz64** [indomus/forum#864] [HUB: Home Assistant]
Scusate ma è normale che il ripristino di HA da backup abbia un DB senza nulla. Funziona tutto, per fortuna, ma lo storico è completamente sparito. Grazie in anticipo.

**Comment by snakuzzo** [comment]
Si vede che era corrotto e lo ha ricreato. Se così fosse, dovrebbe aver salvato un file di backup

**Reply by calz64** [reply]
Scusa l'ignoranza, ma dove lo salva il file di backup?

**Reply by snakuzzo** [reply]
se l'ha salvato, è nello stesso path del file del DB che utilizza...ovvero nella root della directory di configurazione

**Reply by calz64** [reply]
ok grazie, non mi sembra ci sia.
![Cattura](https://user-images.githubusercontent.com/84615011/137522120-377564f3-4342-444f-8afa-67a23375b317.PNG)

**Reply by snakuzzo** [reply]
pazienza
